__tests__/routes/user.test.js

```diff
@@ -51,4 +51,73 @@ describe('users', () => {
     expect(res.status).toBe(401);
     expect(res.body.message).toBe('test@email.com has already been confirmed');
   });
+
+  // Forget password tests
```

Comment:
Like I said. One line comments are unnecessary. Please delete them. I can understand this is a `password reset` test without the comment.

Reply:
Right away!